index.html

```diff
@@ -13,19 +13,39 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <link rel="stylesheet" href="style.css">
     </head>
+
     <body>
```

Comment:
- [ ] The _footer_ is missing in your pages. Kindly ensure to add the footer to the pages.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/133691653-0b90474f-41e3-45d7-9dd5-eb86a4ed7a10.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/130169854-e3b0b337-4389-40f0-b387-cd1587363afe.png)